system/ee/ExpressionEngine/Addons/channel/mod.channel.php

```diff
@@ -663,7 +663,7 @@ private function _generate_field_search_sql($search_fields, $legacy_fields, $sit
                 $fields_sql .= ee()->channel_model->field_search_sql($terms, $search_column_name, $site_id);
             } // foreach($sites as $site_id)
             if (! empty($fields_sql)) {
-                $sql .= 'AND (' . $fields_sql . ')';
+                $sql .= ' AND (' . $fields_sql . ')';
```

Comment:
Space at start of AND here shouldn't be needed, as it's added in before this method is called... Looks like most of the SQL in mod.channel has space at end of appended string as it's building.

Reply:
It is exactly the space that was causing the issue (together with the brackets). But if you have strong objections why it should not be here, please let me know

Reply:
Because this is in a `foreach`, it does need a space before it. Having extra spaces shouldn't be an issue but it does need one either before or after. I'm going to resolve this as I'm fine with this fix.